routing/routing_integration_tests/route_test.cpp

```diff
@@ -26,18 +26,20 @@ namespace
         MercatorBounds::FromLatLon(55.66237, 37.63560), 1700.);
   }
 
-  // Fails because cheackpoints are far from roads (inside Kremlin and inside airport).
-  UNIT_TEST(MoscowToSVOAirport)
-  {
-    integration::CalculateRouteAndTestRouteLength(
-        integration::GetVehicleComponents<VehicleType::Car>(),
-        MercatorBounds::FromLatLon(55.75100, 37.61790), {0., 0.},
-        MercatorBounds::FromLatLon(55.97310, 37.41460), 30470.);
-    integration::CalculateRouteAndTestRouteLength(
-        integration::GetVehicleComponents<VehicleType::Car>(),
-        MercatorBounds::FromLatLon(55.97310, 37.41460), {0., 0.},
-        MercatorBounds::FromLatLon(55.75100, 37.61790), 30470.);
-  }
+// Fails because checkpoints are far from roads (inside Kremlin and inside the airport).
+// This test is commented because the feature, the test is on, will not implement
```

Comment:
```
... because the tested functionality is not implemented.
```.

But what kind of functionality is not implemented now?

Reply:
Right! Everything's ok with this test. I uncommented it and remove the comment.